src/dataconnect/fileUtils.ts

```diff
@@ -110,7 +110,7 @@ export async function pickService(
 // case insensitive exact match indicators for supported app platforms
 const WEB_INDICATORS = ["package.json", "package-lock.json", "node_modules"];
 const IOS_INDICATORS = ["info.plist", "podfile", "package.swift"];
-const ANDROID_INDICATORS = ["androidmanifest.xml", "build.gradle"];
+const ANDROID_INDICATORS = ["androidmanifest.xml", "build.gradle", "build.gradle.kts"];
```

Comment:
IIUC 'build.gradle.kts' is a slightly new equivalent of `build.gradle` (and is what we use in `friendlyeats-android`), so we should check for it too

Reply:
@dconeybe - can you check that we aren't missing anything here?